### ISSUE TITLE
released?

### ISSUE THREAD

**Issue by jasonkarns** [ztombol/bats-assert#3]
Is this considered a 1.0.0 release? Could we get some git tags for releases? (even if not v1.0.0)

(And I strongly recommend the vX.Y.Z form for the tags for reasons)

**Comment by ztombol** [comment]
The libraries will follow [semantic versioning](http://semver.org) so version numbers will be of the form `vX.Y.Z`. In the initial development phase, the major number will be kept `0` and the minor will be bumped on every release. In other words `v0.Y.Z`. I want to release `v1.0.0` after we decided that we reached a certain level of stability.

I didn't want to create a tagged release right after publishing as I thought there would be a fair bit of changes once people could easily try out these libraries (i.e. not in a Bats pull request). Leaving it just a _git_ version would suggest that we don't have a stable API yet. 

In hindsight, it doesn't really matter and I could have tagged `v0.1.0` right away. :)

I will tag `v0.1.0` in `bats-assert` and `bats-core` as soon as I fix #6. So that the package manager support can progress.

**_Edit:**_ _Changed `#5` to `#6`. -- 2016-02-16 21:43 UTC_
